src/helpers/getShowUtils.ts

```diff
@@ -0,0 +1,63 @@
+import { ShowData, TvShowDetailsData, TvShowData, ShowProviders } from '../types/tmdb';
+
+/**
+ * This function is ran after the user enters a name of a TV Show.
+ * @returns {Promise<TvShowData>} | List of shows by searched query.
+ */
+const getShowsByName = async (name: string): Promise<TvShowData> => {
```

Comment:
I think I would prefer `getTvByName`. Even though it doesn't read quite as well, I think:

- shows = movie & TV
- movies = just movies
- tv = just tv

If we have a better name for 'shows' to represent both movie and TV, I am open to that as well. Just want to have 3 distinct labels.

In doing some searching, not sure there is a perfect word for this. `MotionPictureData` doesn't seem great IMO.
<img width="624" alt="image" src="https://user-images.githubusercontent.com/41388783/211038291-016e8439-10d9-4bd6-84b8-a7e251f80ee0.png">

Reply:
@Thenlie Agreed, this was an oversight.